### PR TITLE
GPII-2973:  Adding default RBAC roles, switching Helm to tiller's SA

### PR DESCRIPTION
**Pull request by natarajaya**
@amatas @mrtyler I am currently adding only one cluster-admin role, which absence prevents RBAC-enabled Helm charts to install, but there are more, let's have discussion in 
https://issues.gpii.net/browse/GPII-2973

### CODE REVIEW

File: aws/rakefiles/deploy.rake [hunk 1]
```diff
@@ -80,7 +80,7 @@ task :find_gpii_components => :generate_modules do
 end
 
 desc "Deploy GPII components to cluster"
-task :deploy => [:apply, :configure_kubectl, :wait_for_cluster_up, :init_helm, :find_gpii_components] do
+task :deploy => [:apply, :configure_kubectl, :wait_for_cluster_up, :setup_rbac, :init_helm, :find_gpii_components] do
```
Comment (mrtyler) [inline aws/rakefiles/deploy.rake]: I guess :init_helm should also depend on :setup_rbac.

**Comment by natarajaya** [comment]
Makes sense

File: aws/rakefiles/deploy.rake (hunk 1 above)
Comment (natarajaya) [reply aws/rakefiles/deploy.rake]: Makes sense

**Comment by mrtyler** [comment]
LGTM

**Pull request merged**